apps/vr-tests-react-components/src/stories/DatePickerCompat.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { DatePicker, DateRangeType } from '@fluentui/react-datepicker-compat';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('DatePicker Compat', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .click('.datepicker-input')
+        .snapshot('opened', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>
+  ))
+  .addStory('default', () => <DatePicker input={{ className: 'datepicker-input' }} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  });
+
+storiesOf('DatePicker Compat', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addStory(
+    'DateRange: day',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.Day }} popover={{ open: true }} />,
+    {
+      includeDarkMode: true,
+      includeHighContrast: true,
+    },
+  )
+  .addStory(
+    'DateRange: work week',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.Week }} popover={{ open: true }} />,
+    {
+      includeDarkMode: true,
+      includeHighContrast: true,
+    },
+  )
+  .addStory(
+    'DateRange: week',
+    () => <DatePicker calendar={{ dateRangeType: DateRangeType.WorkWeek }} popover={{ open: true }} />,
```

Comment:
The names of these are swapped:

```suggestion
    'DateRange: week',
    () => <DatePicker calendar={{ dateRangeType: DateRangeType.Week }} popover={{ open: true }} />,
    {
      includeDarkMode: true,
      includeHighContrast: true,
    },
  )
  .addStory(
    'DateRange: work week',
    () => <DatePicker calendar={{ dateRangeType: DateRangeType.WorkWeek }} popover={{ open: true }} />,
```